src/DssSpell.t.sol

```diff
@@ -1103,6 +1103,10 @@ contract DssSpellTest is DSTest, DSMath {
         assertEq(lpFlipEthUsdt.kicks(), 1);
     }
 
+    function testOfficeHoursMatches() public {
+        assertTrue(spell.officeHours() == spellValues.office_hours_enabled);
```

Comment:
I'd prefer to do this check directly in `SpellIsCast` so we keep reduced the amount of tests and it is actually where `expiration_threshold` is also checked.

Reply:
We can move this into the existing test, but it's getting more difficult to track down the particular line of code that the test is failing on.

Reply:
Moved